.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    es6: true,
+  },
+  extends: ['plugin:react/recommended'],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+  },
+  plugins: ['prettier', 'react'],
+  rules: {
+    'import/no-extraneous-dependencies': 'off',
+    'import/prefer-default-export': 'off',
+    'lines-between-class-members': ['error', 'always'],
```

Comment:
we aren't actually writing any classes in this project, are we?

Reply:
No, I didn't notice that. Good catch! I'll remove it.